### PR TITLE
Added post 'Configure AWS Integration'

### PR DESCRIPTION
**Pull request by ValeriaRemneva**
Also all images were uploaded to 'assets/posts_img'.

### CODE REVIEW

File: _posts/2024-07-18-configure-aws-integration.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+![Docker Command](/assets/posts_img/docker_command_btn.png)
+
+![Docker Command](/assets/posts_img/docker_command.png)
+
+A link to the running test will appear in the results panel.
+> Note: Verify that all settings have been made correctly to avoid any issues during the test run.
+
+![Test Results Link](/assets/posts_img/test_results_link.png)
+
+## Next Steps
```
Comment (KarenFlorykian) [inline _posts/2024-07-18-configure-aws-integration.md:95]: Next steps ideally should include instructions on how to use Integration for running distributed tests, for example, with 3 nodes.

**Review by KarenFlorykian — CHANGES_REQUESTED** [review]

File: _posts/2024-07-18-configure-aws-integration.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+---
+
+## Overview
+This guide provides step-by-step instructions for configuring AWS integration in Carrier. AWS integration allows you to leverage Amazon Web Services for your performance testing needs.
+
+### Prerequisites
+Before you begin configuring AWS integration in Carrier, ensure that you have completed the following:
+- Installed and set up Carrier successfully
+- Created a project in Carrier
+- Have valid AWS access keys
```
[the remark this answers is not in the document]
Comment (ValeriaRemneva) [reply _posts/2024-07-18-configure-aws-integration.md]: Added

File: _posts/2024-07-18-configure-aws-integration.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+Follow the steps below to configure AWS integration in Carrier:
+
+#### Step 1: Access the Carrier Web Interface
+1. Open a web browser and enter the URL of your Carrier installation.
+2. Log in to the Carrier web interface using your credentials.
+> Note: Make sure you have logged in using the appropriate user account that has access to the project where you want to configure AWS integration.
+
+#### Step 2: Create AWS Secret Access Key in Secrets Section
+1. Navigate to **Configuration** > **Secrets**.
+2. Create a new secret with the name `<aws_secret_access_key>` and provide the necessary AWS secret access key.
```
[the remark this answers is not in the document]
Comment (ValeriaRemneva) [reply _posts/2024-07-18-configure-aws-integration.md]: Updated

File: _posts/2024-07-18-configure-aws-integration.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+![Create AWS Secret](/assets/posts_img/aws_secret.png)
+
+#### Step 3: Integrate with Amazon in the Clouds Block
+1. Navigate to **Integrations** > **Clouds**.
+2. Select integration with Amazon.
+
+![Amazon Integration](/assets/posts_img/aws_integration.png)
+
+#### Step 4: Fill in Required Fields and Update
+1. Fill in the required fields such as `Access Key`, `Secret Access Key`, and `Region`.
```
[the remark this answers is not in the document]
Comment (ValeriaRemneva) [reply _posts/2024-07-18-configure-aws-integration.md]: Added

File: _posts/2024-07-18-configure-aws-integration.md (hunk 1 above)
Comment (ValeriaRemneva) [reply _posts/2024-07-18-configure-aws-integration.md:99]: Corrected this part. Instructions on how to use Integration for running distributed tests I think should be in the next post.

**Review by KarenFlorykian — APPROVED** [review]

**Pull request merged**